services/web/server/src/simcore_service_webserver/studies_dispatcher/_projects.py

```diff
@@ -169,4 +169,7 @@ async def add_new_project(app: web.Application, project: Project, user: UserInfo
     assert _project_db["uuid"] == str(project.uuid) # nosec
 
     # This is a new project and every new graph needs to be reflected in the pipeline db
+    #
+    # TODO: Ensure this user has access to these services!
+    #
```

Comment:
actually this check should be done before no? how are we going to tackle logged-in vs guest user for such use cases?